packages/database/src/migrations/20210416052750-DailyCasesTrendGrahphPalauFacilityLevel-modifies-data.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+import { insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const THINGS = {
+  AFR: {
+    dataElement: 'PSSS_AFR_Daily_Cases',
+    name: 'Acute fever and rash',
+    color: '#F0965BFF', // orange
+  },
+  Diarrhoea: {
+    dataElement: 'PSSS_DIA_Daily_Cases',
+    name: 'Diarrhoea',
+    color: '#81DEE4FF', // aqua
+  },
+  ILI: {
+    dataElement: 'PSSS_ILI_Daily_Cases',
+    name: 'Influenza like illness',
+    color: '#4DA347FF', // green
+  },
+  PF: {
+    dataElement: 'PSSS_PF_Daily_Cases',
+    name: 'Prolonged fever',
+    color: '#1C49A7FF', // blue
+  },
+  DLI: {
+    dataElement: 'PSSS_DLI_Daily_Cases',
+    name: 'Dengue like illness',
+    color: '#8455F6', // purple
+  },
+  Conjunctivitis: {
+    dataElement: 'PSSS_CON_Daily_Cases',
+    name: 'Conjunctivitis',
+    color: '#BE72E0', // pink
+  },
+};
+
+const DASHBOARD_GROUP_TEMPLATE = {
+  organisationLevel: 'Facility',
+  userGroup: 'PSSS Tupaia',
+  organisationUnitCode: 'NR',
```

Comment:
I thought this dashboard is for Palau, but isn't NR stands for Nauru?

Reply:
Thanks Chris! Good spot :pray: